app/assets/javascripts/actions/task.js

```diff
@@ -0,0 +1,46 @@
+import actionTypes from './actionTypes';
+
+export const createTaskSuccess = (task, story) => ({
+  type: actionTypes.ADD_TASK,
+  task,
+  story
+});
+
+export const deleteTaskSuccess = (task, story) => ({
+  type: actionTypes.REMOVE_TASK,
+  task,
+  story
+});
+
+export const updateTaskSuccess = (task, story) => ({
+  type: actionTypes.UPDATE_TASK,
+  task,
+  story
+});
+
+export const postTask = (projectId, story, newAttributes) => {
```

Comment:
I think here like projectId, you can pass only storyId to you params since [here](https://github.com/Codeminer42/cm42-central/pull/440/files#diff-069c4a0c472d216643d1774dbeec3846R27) is the only attribute that you are using.

Reply:
So here, I use all of the params because they pass [https://github.com/Codeminer42/cm42-central/pull/440/files#diff-4683d203f1fd881b51e0d93c7105d8fcR23](herel) to make a ajax request